docs/components/sliders.md

```diff
@@ -0,0 +1,393 @@
+<!--docs:
+title: "Sliders"
+layout: detail
+section: components
+excerpt: "Sliders allow users to make selections from a range of values."
+iconId: slider
+path: /catalog/sliders/
+-->
+
+# Slider
+
+[Sliders](https://material.io/components/sliders/) allow users to make
+selections from a range of values.
+
+!["Slider with sound icon buttons on each end."](assets/sliders/sliders_hero.png)
+
+## Contents
+
+- [Using sliders](#using-sliders)
+- [Continuous slider](#continuous-slider)
+- [Discrete slider](#discrete-slider)
+- [Theming sliders](#theming-sliders)
+
+## Using sliders
+
+Sliders reflect a range of values along a bar, from which users may select a single value. They are ideal for adjusting settings such as volume, brightness, or applying image filters.
+
+Sliders can have icons on both ends of the bar that reflect a range of values.
+
+### Making sliders accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+## Types
+
+There are two types of sliders:
+
+1. [Continuous slider](#continuous-slider)
+1. [Discrete slider](#discrete-slider)
+
+!["Slider examples of both continuous and discrete sliders."](assets/sliders/sliders_types.png)
+
+A slider with one thumb is called a single point slider, and a slider with two thumbs is called a range slider.
+
+### Anatomy and key properties
+
+A slider has a track, one or two thumbs, and optional value label. A discrete slider also has tick marks.
+
+![Slider anatomy diagram](assets/sliders/sliders_anatomy.png)
+
+1. Track
+1. Thumb
+1. Value label (optional)
+1. Tick mark (discrete sliders)
+
+#### Track attributes
+
+| &nbsp; | Attribute |
+|---|---|
+| Track Shape | Wrap your `Slider` widget with a `SliderTheme` widget within the data parameter use the `SliderThemeData` widget. Within this widget set the `trackShape` parameter. |
```

Comment:
nit: The first sentence here and below is a bit hard to understand. It could be rephrased, perhaps to something like:

Wrap your `Slider` widget with a `SliderTheme` widget. Use the `SliderThemeData` widget for the `SliderTheme.data` parameter with the `SliderThemeData.trackShape` parameter set.